libs/common/src/lib/widgets/widget-generator.component.html

```diff
@@ -63,6 +63,17 @@ <h3>{{ 'stash.widgets.configuration' | translate }}</h3>
             formControlName="filter">
             {{ 'stash.widgets.filter' | translate }}
           </pa-toggle>
+          <div class="filter-type">
+            <pa-select
+              *ngIf="mainFormFeatures.filter"
+              [value]="filters"
+              (valueChange)="onFiltersChange($event)"
+              [label]="'stash.widgets.filter_type' | translate">
+              <pa-option value="both">{{ 'stash.widgets.labels_entities' | translate }}</pa-option>
```

Comment:
I think you should not have an option `both` because it's implying we will always have only two filter options, which is probably not true.
Better than a select, you should probably use a checkbox list (one checkbox by filter type, so for now only two) with all checkboxes selected by default

Reply:
ok, I'll try using checkboxes